crates/napi/src/next_api/endpoint.rs

```diff
@@ -180,9 +181,10 @@ pub fn endpoint_server_changed_subscribe(
         func,
         move || {
             async move {
-                subscribe_issues_and_diags(endpoint, issues)
-                    .strongly_consistent()
-                    .await
+                let operation = subscribe_issues_and_diags(endpoint, issues);
+                let result = operation.strongly_consistent().await?;
+                apply_effects(operation).await?;
```

Comment:
Is there a problem with doing this after the `strongly_consistent` read, instead of during? I.e. If operation got invalidated immediately after the strongly consistent read and began re-execution, could `apply_effects` read the wrong thing?

It seems like [`strongly_consistent_catch_collectables`](https://github.com/vercel/next.js/blob/c3bc7d05983d3c5604d4532ceb706d5973aef642/crates/napi/src/next_api/endpoint.rs#L101) in `subscribe_issues_and_diags` would have this issue too.

Assuming this is a problem, I think it's okay to ignore it for now, as it's very unlikely to happen, but it should be documented as a comment.

Reply:
Yeah there is a tiny chance of that. Also similar for get_issues. I'm aware and we should probably do something about that eventually. Probably need a strongly consistent read, which also reads the collectibles at the same time.

Reply:
Let's just add a comment?

Reply:
I'll improve it in a follow-up PR